client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -242,16 +242,20 @@ protected Integer doBackward(String s) {
      */
     protected static class ListStringifier<T> extends Stringifier<List<T>> {
 
+        private static final String DEFAULT_DELIMITER = ",";
+
         private final Class<T> listGenericClass;
 
         /**
-         * The delimiter for the passed elements in the String representation. Comma by default.
+         * The delimiter for the passed elements in the String representation,
+         *  {@code DEFAULT_DELIMITER} by default.
          */
-        private String delimiter = ",";
+        private final String delimiter;
 
-        public ListStringifier(Class<T> listGeneric) {
+        public ListStringifier(Class<T> listGenericClass) {
             super();
-            this.listGenericClass = listGeneric;
+            this.delimiter = DEFAULT_DELIMITER;
+            this.listGenericClass = listGenericClass;
         }
 
         /**
```

Comment:
Please create a valid documentation for the 
`public ListStringifier(Class<T> listGenericClass, String delimiter)` constructor.